doc/md/knowledge/postgres/partial-indexes.md

```diff
@@ -0,0 +1,306 @@
+---
+id: partial-indexes
+title: Partial Indexes in PostgreSQL
+slug: /knowledge/postgres/partial-indexes
+---
+
+### Overview of Partial Indexes
+
+#### What are Partial Indexes?
+
+With PostgreSQL we can create partial indexes to improve performance and reduce the cost, all while minimizing the amount of space used to store it on disk.
```

Comment:
```suggestion
With PostgreSQL, users may create partial indexes to improve performance and reduce the cost, all while minimizing the amount of space used to store it on disk.
```
I prefer a bit "drier" tone here. When we speak as we in the docsite we usually refer to ourselves (the atlas contributors etc).